ios/Classes/SwiftFpjsProPlugin.swift

```diff
@@ -14,38 +14,57 @@ public class SwiftFpjsProPlugin: NSObject, FlutterPlugin {
     public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
         if (call.method == "init") {
             if let args = call.arguments as? Dictionary<String, Any>,
-               let token = args["apiToken"] as? String {
-                let region = args["region"] as? String
+                let token = args["apiToken"] as? String {
+                let region = parseRegion(passedRegion: args["region"] as? String, endpoint: args["endpoint"] as? String)
 
-                var endpoint: URL?
-                if let endpointString = args["endpoint"] as? String {
-                    endpoint = URL(string: endpointString)
-                }
-
-                initFpjs(token: token, endpoint: endpoint, region: region)
+                initFpjs(token: token, region: region)
                 result("Successfully initialized FingerprintJS Pro Client")
             } else {
                 result(FlutterError.init(code: "errorApiToken", message: "missing API Token", details: nil))
             }
         } else if (call.method == "getVisitorId") {
             let args = call.arguments as? Dictionary<String, Any>
-            let tags = args?["tags"] as? FingerprintJSProClient.Tags
+            let tags = args?["tags"] as? [String: JSONTypeConvertible]
```

Comment:
I wonder whether this type conversion should really fail silently. If the developer passes anything that doesn't convert to `JSONTypeConvertible`, the entire `tags` dictionary is thrown away without any indication.